docker-compose.yml

```diff
@@ -0,0 +1,60 @@
+version: "3.4"
+
+services:
+  server:
+    build:
+      context: ./backend
+      dockerfile: Dockerfile
+    volumes:
+      - ./backend/:/backend/
+    entrypoint: ./docker-start.sh
+    container_name: taia-x-backend
+    env_file:
+      - ./backend/.env
+    ports:
+      - 5678:5678
+      - 8000:8000
+    depends_on:
+      - db
+      - ipfs
+
+  frontend:
+    build:
+      context: ./frontend
+      dockerfile: Dockerfile
+    volumes:
+      - ~/.composer-docker/cache:/root/.composer/cache:delegated
+      - ./frontend/:/opt/taia-x/frontend:rw,delegated
```

Comment:
@amrap030, just for me to understand, what does `:rw,delegated` at the end of the volume path do?

Reply:
It mounts the volume as `read-write` and `delegated` is used when docker container performs changes and host is in read only mode. I noticed, that this makes no sense and `cached` instead of `delegated` is the better option. `cached` is used when the host performs changes and the container is in read only mode. I also removed `rw`.